src/webgl-context/context-state.js

```diff
@@ -23,7 +23,7 @@ import assert from '../utils/assert';
 //   in an array, in a form that can be accepted by the setter.
 export const LUMA_SETTERS = {
   framebuffer: (gl, framebuffer) => {
-    const handle = framebuffer && framebuffer.handle ? framebuffer.handle : framebuffer;
+    const handle = framebuffer && 'handle' in framebuffer ? framebuffer.handle : framebuffer;
```

Comment:
Add a comment on when `handle` would not exist?

Reply:
I think when `handle` itself is passed to the function as `framebuffer`, backward compatibility? CC: @ibgreen .